### PR TITLE
Formatting Create, Delete, ParseOrders, and ProcessOne

### PR DESCRIPTION
**Pull request by KyleZim97**
I formatted these classes to googles formatting guide pretty much.

- The SQL queries are formatted more for readability than the google formatting guide.
- Created comments for code that was not commented 
- Added brief class descriptions for the classes just to explain them better
- Fixed spelling and organization of these classes.
- Tested to make sure they still run and they do!

### CODE REVIEW

File: app/src/main/java/com/yagni/model/Create.java [hunk 1]
```diff
@@ -3,30 +3,44 @@
...
         // Inserting into the database
-        // Try catches SQLExecption
+        // Try catches SQLException
         try {
             // Creates a statement object
             Statement statement = linkDB.getConnection().createStatement();
             // Calling the execute method to execute an INSERT statement
-            statement.execute("INSERT INTO `product`(product_id, quanity, Whole_sale, Sale_cost, vendor_id) VALUES ('"+productId+"',"+quantity+","+wholeSale+","+salePrice+",'"+supplierId+"')");
+            statement.execute("INSERT INTO `product`(product_id, quanity, Whole_sale, Sale_cost, vendor_id) "
+                            + " VALUES ('"+productId+"',"+quantity+","+wholeSale+","+salePrice+",'"+supplierId+"')");
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/Create.java:39]: nice line-wrap. Looks like you hit tab 4 times on it though. It should be 2 tabs past line 38. Alyssa has an example of how it should look on Line 362&363 under ManageInvGUI.java

File: app/src/main/java/com/yagni/model/Delete.java [hunk 2]
```diff
@@ -3,26 +3,35 @@
...
 
+/*
+* This class Deletes a new Item in the inventory
+*/
 public class Delete {
 
     private static String deleteID;
 
-    public Delete (String deleteIDin){
+    // CONSTRUCTORR!!!!!!!!!
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/Delete.java:13]: LMAO

File: app/src/main/java/com/yagni/model/ProcessOne.java [hunk 3]
```diff
@@ -67,68 +81,80 @@ public ProcessOne(){
...
-        //Use an update statement to update the products table in the database
-        statement.execute("UPDATE `yagni_inv_db`.`product` SET `quanity` = '" + updatedStock + "' WHERE (`product_id` = '" + productId +"');");
-
-        }catch (SQLException e){
+        try {
+            Statement statement = linkDb.getConnection().createStatement();
+
+            // Calling the executeQuery method to execute a statement to the database
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM yagni_inv_db.product" 
+                                                        + " WHERE product_id = '" + productId + "';");
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ProcessOne.java:97]: same issue here with the line-wrap. Just hit tab twice for the indent. Think you have like 6 tabs

File: app/src/main/java/com/yagni/model/Create.java (hunk 1 above)
Comment (S4kRiPp4) [reply app/src/main/java/com/yagni/model/Create.java:39]: > nice line-wrap. Looks like you hit tab 4 times on it though. It should be 2 tabs past line 38. Alyssa has an example of how it should look on Line 362&363 under ManageInvGUI.java

I seen what you did here and I would be fine w/o the changes.

File: app/src/main/java/com/yagni/model/ProcessOne.java (hunk 3 above)
Comment (S4kRiPp4) [reply app/src/main/java/com/yagni/model/ProcessOne.java:97]: > same issue here with the line-wrap. Just hit tab twice for the indent. Think you have like 6 tabs

can keep this the same too, ignore comment

File: app/src/main/java/com/yagni/model/ProcessOne.java [hunk 4]
```diff
@@ -67,68 +81,80 @@ public ProcessOne(){
...
+            intStock = intStock - intQuantity;
+            updatedStock = String.valueOf(intStock);
+
+            // Use an update statement to update the products table in the database
+            statement.execute("UPDATE `yagni_inv_db`.`product` SET `quanity` = '" + updatedStock 
+                            + "' WHERE (`product_id` = '" + productId + "');");
+
+        } catch (SQLException e) {
             System.out.println(e);
         }
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ProcessOne.java]: delete whitespace 116

**Review by S4kRiPp4 — APPROVED** [review]
Thanks for changes, nice in-depth comments.

File: app/src/main/java/com/yagni/model/Create.java (hunk 1 above)
Comment (KyleZim97) [reply app/src/main/java/com/yagni/model/Create.java:39]: Yeah I wanted line it up so it looked more like an SQL statement for easier readability.

**Review by chillyssa — APPROVED** [review]
Nice Kyle! I like the choice to line wrap Sql statements so they look like actual SQL structure. 

Also, thanks for updating the classes to use variable declarations and constructors as java standards use. This unifies our code a lot! 

Looks good to me... @S4kRiPp4 we all good with the whitespace??

**Pull request merged**